tracee-ebpf/tracee/vmlinux/x86_64/vmlinux-arch.h

```diff
@@ -0,0 +1,65 @@
+#ifndef __VMLINUX_ARCH_H__
+#define __VMLINUX_ARCH_H__
+
+#include <vmlinux-core.h>
+#include <vmlinux-flavored.h>
+
+#ifndef BPF_NO_PRESERVE_ACCESS_INDEX
+#pragma clang attribute push (__attribute__((preserve_access_index)), apply_to = record)
+#endif
+
+struct thread_info {
+	u32 status;
+};
+
+struct task_struct {
+	struct thread_info         thread_info;
+	unsigned int               flags;
+	short unsigned int         migration_flags;
+	struct mm_struct *         mm;
+	int                        exit_code;
+	long unsigned int          atomic_flags;
+	pid_t                      pid;
+	pid_t                      tgid;
+	struct task_struct *       real_parent;
+	struct task_struct *       group_leader;
+	struct pid *               thread_pid;
+	struct list_head           thread_group;
+	const struct cred  *       real_cred;
+	char                       comm[16];
+	struct files_struct *      files;
+	struct nsproxy *           nsproxy;
+	unsigned int               sas_ss_flags;
+	unsigned int               psi_flags;
+	struct css_set *           cgroups;
+};
```

Comment:
There are several fields here that are not used by tracee, do we really need them?
migration_flags,atomic_flags,sas_ss_flags,psi_flags

Reply:
nope, they are leftovers from the process I used to generate the slim version. Im cleaning that up.